browser-test/src/applicant/northstar_application_download.test.ts

```diff
@@ -0,0 +1,511 @@
+        )
+        expect(demographicsCsvContent).toContain(
+          'sarah,,smith,,NOT_SELECTED,NOT_SELECTED,NOT_SELECTED,SELECTED,1000.00,05/10/2021,op2_admin,',
+        )
+      } else {
+        expect(demographicsCsvContent).toContain(
+          'Opaque ID,Program,Submitter Type,TI Email (Opaque),TI Organization,Create Time,Submit Time,Status,sample name question (first_name),sample name question (middle_name),sample name question (last_name),sample name question (suffix),csvcolor (selections - red_admin),csvcolor (selections - green_admin),csvcolor (selections - orange_admin),csvcolor (selections - blue_admin),csvcurrency (currency),csvdate (date),dropdowncsvdownload (selection),numbercsvdownload (number)',
+        )
+        expect(demographicsCsvContent).toContain(
+          'sarah,,smith,,1000.00,05/10/2021,op2_admin',
+        )
+      }
+
+      await adminQuestions.createNewVersion('Sample Name Question')
+      await adminQuestions.exportQuestionOpaque('Sample Name Question')
+      await adminPrograms.publishProgram(programName)
+
+      await adminPrograms.gotoAdminProgramsPage()
+      const newDemographicsCsvContent = await adminPrograms.getDemographicsCsv()
+
+      if (demographicsCsvContent.includes('Status')) {
+        expect(newDemographicsCsvContent).toContain(
+          'Opaque ID,Program,Submitter Type,TI Email (Opaque),TI Organization,Create Time,Submit Time,Status,csvcolor (selections - red_admin),csvcolor (selections - green_admin),csvcolor (selections - orange_admin),csvcolor (selections - blue_admin),csvcurrency (currency),csvdate (date),dropdowncsvdownload (selection),numbercsvdownload (number),sample name question (first_name),sample name question (middle_name),sample name question (last_name),sample name question (suffix)',
+        )
+      } else {
+        expect(newDemographicsCsvContent).toContain(
+          'Opaque ID,Program,Submitter Type,TI Email (Opaque),TI Organization,Create Time,Submit Time,csvcurrency (currency),csvdate (date),dropdowncsvdownload (selection),numbercsvdownload (number),sample name question (first_name),sample name question (middle_name),sample name question (last_name),sample name question (name_suffix)',
+        )
+      }
+      expect(newDemographicsCsvContent).not.toContain(',sarah,,smith')
+      expect(newDemographicsCsvContent).toContain(',op2_admin,')
+      expect(newDemographicsCsvContent).toContain(',1600,')
+
+      if (isLocalDevEnvironment()) {
+        // The hashed values "sarah", empty value, "smith", with the dev secret key.
+        expect(newDemographicsCsvContent).toContain(
+          '5009769596aa83552389143189cec81abfc8f56abc1bb966715c47ce4078c403,057ba03d6c44104863dc7361fe4578965d1887360f90a0895882e58a6248fc86,6eecddf47b5f7a90d41ccc978c4c785265242ce75fe50be10c824b73a25167ba',
+        )
+      }
+    })
+
+    test('download finished application', async ({
```

Comment:
suggestion (blocking): I see the issue. "Application download" means two different things, and tests for both flows got added to this file. This test is the only one that tests the applicant flow, so it is the only one that needs to be migrated. We should split up the tests in the old file as well so it's clear they're testing different things. I can make an issue for that.

Reply:
filed https://github.com/civiform/civiform/issues/9373  to split up the test cases and move admin test cases to .../admin folder.  assigned to myself